src/lib.rs

```diff
@@ -303,19 +310,19 @@ mod candle_auction {
             }
         }
 
-        /// Pluggable reward logic: OPTION-1.    
```

Comment:
All these double-spaces in line endings are put there intentionally for formatting objectives: as in markdown syntax they force a newline (like `\n`). Just to make it look a little more prettier in final cargo doc. I'd like to leave them therefore (unless such a formatting violates any guidelines, does it?)

Reply:
Ah my bad! I wasn't aware of this formatting magic :) thanks for pointing it out